sacro/views.py

```diff
@@ -0,0 +1,5 @@
+from django.template.response import TemplateResponse
+
+
+def index(request):
+    return TemplateResponse(request, "index.html")

```

Comment:
If we're only going to have this return a template, I'd be tempted to just put it in the urls.py directly with `TemplateView.as_view(template_name="index.html")`.

Reply:
So, that's all it does right now.  It might do something different soon?